tests/test_api_integration.py

```
@@ -0,0 +1,469 @@
+from unittest.mock import MagicMock
+
+# Re-patch discord after conftest to ensure Entitlement exists for api imports
+_discord_mock = MagicMock()
+_discord_mock.Entitlement = MagicMock()
+sys.modules["discord"] = _discord_mock
+sys.modules["discord.ext"] = MagicMock()
+sys.modules["discord.ext.commands"] = MagicMock()
+sys.modules["discord.app_commands"] = MagicMock()
+
+import api
+from api import get_table_definitions
```

Flake8 (7.3.0)

[error] 'api.get_table_definitions' imported but unused

(F401)